packages/frontend/src/renderer/containers/widgets/joinCommunity/joinCommunity.basic.test.tsx

```diff
@@ -126,65 +126,40 @@ describe('join community', () => {
     expect(createUsernameTitle).toBeVisible()
   })
 
-  it('user rejoins to remembered community with certificate', async () => {
+  it('user rejoins to remembered community with user certificate', async () => {
     const factoryStore = (await prepareStore()).store
     const factory = await getFactory(factoryStore)
 
     const community = await factory.create<
       ReturnType<typeof communities.actions.addNewCommunity>['payload']
     >('Community')
+    await factory.create<
+      ReturnType<typeof identity.actions.addNewIdentity>['payload']
+    >('Identity', { id: community.id, nickname: 'alice1' })
 
-    const identityAlpha: Identity = {
-      id: community.id,
-      nickname: 'nickname',
-      hiddenService: {
-        onionAddress: '',
-        privateKey: ''
-      },
-      dmKeys: {
-        publicKey: '',
-        privateKey: ''
-      },
-      peerId: {
-        id: '',
-        pubKey: '',
-        privKey: ''
-      },
-      userCsr: null,
-      userCertificate: ''
-    }
+    factoryStore.dispatch(socketActions.setConnected())
 
-    const { store } = await prepareStore({
-      [StoreKeys.Socket]: {
-        ...new SocketState(),
-        isConnected: true
-      },
-      [StoreKeys.Modals]: {
-        ...new ModalsInitialState(),
-        [ModalName.joinCommunityModal]: { open: true }
-      },
-      [NectarStoreKeys.Communities]: factoryStore.getState().Communities,
-      [NectarStoreKeys.Identity]: {
-        ...new identity.State(),
-        identities: identityAdapter.setAll(
-          identityAdapter.getInitialState(),
-          [identityAlpha]
-        )
-      }
-    })
+    factoryStore.dispatch(modalsActions.openModal({
+      name: ModalName.joinCommunityModal
+    }))
+
+    factoryStore.dispatch(identity.actions.storeUserCertificate({
+      userCertificate: '',
+      communityId: community.id
+    }))
 
     const result1 = renderComponent(
       <>
         <JoinCommunity />
         <CreateUsernameModal />
         <LoadingPanelModal />
       </>,
-      store
+      factoryStore
     )
     const switchLink1 = result1.queryByText(LoadingMessages.CreateCommunity)
     expect(switchLink1).not.toBeNull()
```

Comment:
![image](https://user-images.githubusercontent.com/61462655/152016342-b6f234b8-938f-4f29-810f-9d352ad5e380.png)

Reply:
i think you are wrong, when i changed only userCertificate in store it recvied NULL

Reply:
What do you mean by 'removed only userCertificate', I don't get it? 
What I suggest to use instead of `.not.toBeNull` is '.toBeInTheDocument' or '.toBeVisible'

Reply:
i mean when user certificate is in store queryByText(...) method will receive null and when we remove that certificate from store it will receive our Loading Component :p in case that u said that you think 'this assertion will always be true'

Reply:
but u right, .toBeInTheDocument or .toBeVisible is better idea